backend/cmm_helpers.ml

```diff
@@ -1000,7 +1069,18 @@ let machtype_identifier t =
   in
   String.of_seq (Seq.map char_of_component (Array.to_seq t))
 
-let unique_arity_identifier arity =
+let extended_machtype_identifier t =
```

Comment:
What is this function for?

Reply:
Left-over from development, will delete.